.changeset/neat-parents-search.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/connectivity': minor
+---
+
+[New Functionality] Add transform function for OAuth2ClientCredentials destinations.
```

Comment:
[q] Maybe a bit more context here? Like this is for custom service binding types that will be transformed to an OAuth2ClientCredentials destination.